src/services/UserService.ts

```
@@ -19,4 +19,15 @@ export class UserService {
     if (!res.ok) throw new Error(`Failed to fetch experience: ${res.status}`);
     return res.json();
   }
+
+  static async getUserCertification(userId: string, token: string) {
+    console.log(token);
+    console.log(userId);
+    const res = await fetch(`${this.BASE_URL}/api/certificates/user/${userId}`, {
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)